core/scss/components/alert/module.scss

```diff
@@ -1,12 +1,34 @@
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../icon/settings';
-@import '../list-item/settings';
-@import '~@momentum-ui/icons/scss/mixins';
-@import '../../tools/mixins/flex';
+// @import '../../tools/functions/core';
+@import '../../tools/mixins/core';
+
+// Settings
+@import '../../settings/settings';
+
+// Tools
+@import '../../tools/mixins/baseline';
 @import '../../tools/mixins/border-radius';
+@import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus';
+@import '../../tools/mixins/grid';
+@import '../../typography/mixins';
+
+// Component Helpers
+@import '../../components/button/mixins';
+@import '../../components/button/button';
+
+@import '../../generic/global';
+@include brand-font-family-ciscoFont;
+
+// Alert Specific
+@mixin icon-info_24() { content: char-from-code("f5af"); }
```

Comment:
These mixins should already be defined in the [icon library mixins](https://github.com/momentum-design/momentum-ui/blob/master/icons/scss/mixins.scss). Import them from `@import '~@momentum-ui/icons/scss/mixins';` instead. If you recreate them here, the char-codes could/will change when new icons are added to the library.

Reply:
thank you, this will fit in many other places